Resources/public/Admin.js

```diff
@@ -329,6 +329,11 @@ var Admin = {
 
     setup_collection_buttons: function(subject) {
 
+        // Remove previously attached event(s) which causing function to be called multiple times
+        jQuery(subject).on('click', '.sonata-collection-add', function(event){
+            return false;
+        });
+
```

Comment:
Because the function "setup_collection_buttons" called more than once, the onclick will be triggered more than once.
So I am adding this line to disable any previously registered events in order for the click event to be triggered once. Please refer to the issue to see the problem I had.

Here is console output ( *I added* "adding click on ...") to log what this function delegated for.

[Sonata.Admin] [Admin] apply shared_setup
**sonata_admin_js.js:95 adding click on  #document**
sonata_admin_js.js:93 [Sonata.Admin] [object Object]
sonata_admin_js.js:93 [Sonata.Admin] [Admin] apply shared_setup
**sonata_admin_js.js:95 adding click on  [div.page-composer__dyn-content, prevObject: bH.fn.init[1], context: document, selector: ".page-composer__dyn-content"]**
sonata_admin_js.js:93 [Sonata.Admin] [object Object]
sonata_admin_js.js:93 [Sonata.Admin] [Admin] apply shared_setup
**sonata_admin_js.js:95 adding click on  [div.page-composer__container__child__content, prevObject: bH.fn.init[1], context: li.page-composer__container__child, selector: ".page-composer__container__child__content"]0: div.page-composer__container__child__contentcontext: li.page-composer__container__child.page-composer__container__child--expandedlength: 1prevObject: bH.fn.init[1]selector: ".page-composer__container__child__content"__proto__: bH[0]**
sonata_admin_js.js:93 [Sonata.Admin] [object Object]